modules/upgrade/pages/migrate/data-migration.adoc

```diff
@@ -65,8 +61,9 @@ cat << EOF > mm2.properties
 clusters = <cluster_name_1>, <cluster_name_2>
 
 // Assign IP addresses to the cluster names
-<cluster_name_1>.bootstrap.servers = <cluster_1>_cluster_ip>:9092
-<cluster_name_2>.bootstrap.servers = <cluster_2>_cluster_ip>:9092
+// <cluster>.bootstrap.servers may also be a comma-separated list of bootstrap hosts, if the cluster has several
```

Comment:
I don't think the comment is necessary, since bootstrap.servers is plural and defined as a list. But @hcoyote please confirm! Also, my confusing comment about `<cluster_1_ip>` is because I thought bootstrap.servers required both cluster name and cluster IP (and I wanted `<cluster_name_1>` and `<cluster_1>` to match, so if a user edited that code in the doc, the name would get edited in both places). But apparently only IP is required!

Reply:
as far as I know, it's a comma separated list of bootstrap addresses (name or ip), so it doesn't need to have both the name and ip address of each host.

in this case, cluster_name_1 is a logical name for the mm2 config to refer to in various parts of the config, it would not be a host name